shared/api/storage_pool_volume.go

```diff
@@ -60,6 +60,12 @@ type StorageVolumePost struct {
 	//
 	// API extension: storage_api_remote_volume_snapshots
 	VolumeOnly bool `json:"volume_only" yaml:"volume_only"`
+
+	// Project for the source volume
+	// Example: project1
+	//
+	// API extension: storage_api_project
+	Project string `json:"project,omitempty" yaml:"project,omitempty"`
```

Comment:
Hmm, I don't think we should need this one.

A pool storage volume is always queried through a specific project. We don't have any cases where we return this struct across projects.